src/geofabrics/lidar_fetch.py

```diff
@@ -10,12 +10,18 @@
 import requests
 import boto3
 import botocore
+import botocore.client
 import typing
 from . import geometry
 
 
 class OpenTopography:
     """ A class to manage fetching LiDAR data from Open Topography
```

Comment:
@Pooja3894 this file might be of interest to you? It is one of the files I will migrate to its own repository later in the week.